packages/server/src/meta/table.ts

```diff
@@ -0,0 +1,13 @@
+import { Knex } from 'knex'
+import { Table } from '../base/table'
+
+export class MetaTable extends Table {
+  get id() {
+    return 'msg_meta'
+  }
+
+  create(table: Knex.CreateTableBuilder) {
+    table.timestamp('time').primary()
```

Comment:
It's not absolutely necessary but it creates an index at the same time, which we use later to get the latest data. It doesn't change anything since the amount of rows is so low, but I don't see any reason to remove it either.